### DISCUSSION TITLE
Can we use the kubernetes dashboard alpha version in production

### DISCUSSION THREAD

**Discussion by irparajababu** [kubernetes/dashboard#8247] [General]
Hi,

Shall I use the below kubernates dashboard in production. 

We are using kubernetes version 1.23.16.

https://github.com/kubernetes/dashboard/releases/tag/v3.0.0-alpha0
kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v3.0.0-alpha0/charts/kubernetes-dashboard.yaml

**Comment by floreks** [comment]
Kubernetes `v1.23` is outdated and we can't guarantee full compatibility. The last Dashboard release with full compatibility was `v2.5.1`. Newer versions do not guarantee full compatibility meaning some resources might not be able to load due to API version mismatch, but it can also be fully functional.

In general, the alpha in the release is not related to the Dashboard app itself because feature-wise nothing really changed but rather to the deployment process of the application and making the helm chart flexible and secure enough to cover the most popular use cases.

The application itself is stable and if you do not require any custom setup then you can consider it a stable version.
